### PR TITLE
Fix brand page

### PR DESCRIPTION
**Pull request by NebuDev14**
The text on the navbar for the brands page blended in with the background of the entire website, so I changed the color to make it readable  I also noticed that the "Phantom Sans is our brand font" didn't have any spacing between them so I added that as well!

Navbar
Text

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/brand**](https://v3-kzcn8tpdh.hackclub.dev/brand) |
|:-:|
|<a href="https://v3-kzcn8tpdh.hackclub.dev/brand">Screenshot of /brand</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/XdYxRgUz9miqBrTimEUBwmbNih8SQS9BZPpfLCvWJJf_H_VWxyKSUD7dW4n7K6Lva3ZoHiGyJ81bY9bwbRv1LA.png">(view full size)</a> |







Commit 9f61c98c2c164c2ab0fa7ba56cbdf11706fc4e1c (https://v3-kzcn8tpdh.hackclub.dev).

**Review by sampoder — CHANGES_REQUESTED** [review]
Could we fix it on dark mode?

Screenshot 2021-08-22 at 9 36 12 AM

File: pages/brand.js [hunk 1]
```diff
@@ -99,6 +99,8 @@ const ColorSwatch = ({ bg }) => (
   </Card>
 )
 
+const color = '#000000'
```
Comment (sampoder) [inline pages/brand.js:102]:
```suggestion
const color = 'text'
```
I just want to see if this will work

Comment (sampoder) [reply pages/brand.js]: This didn't work :(

**Comment by NebuDev14** [comment]
Let me see if I can pass in the hack club theme ui variable colors so that it changes on dark mode!

**Comment by NebuDev14** [comment]
Does this work? I can't find the toggle dark mode button on the hack club website

**Review by sampoder — APPROVED** [review]
Fixed the nav bar in another branch, thanks for bring this up and the fix on the font title!

**Pull request merged**